frontend/src/types/profile.ts

```diff
@@ -13,9 +14,21 @@ export interface ProfileRunnerPost {
   reviewStatus: ReviewStatus;
 }
 
+export interface RunnerProfileResponse extends Profile {}
+
+export interface SupporterProfileResponse extends Profile {}
+
+export interface RunnerProfileRequest extends ProfileRequest {}
```

Comment:
```suggestion
export interface GetSupporterProfileResponse extends Profile {}

export interface PatchRunnerProfileRequest extends ProfileRequest {}
```
컨벤션에 맞춰 http 메서드명을 붙이고 위에 있는 기존 타입은 지우는게 어떨까요?
마이페이지에서 사용할 타입은 따로 만들어야 할 것 같아요.